### PR TITLE
WFLY-5580 Add warning for migration of proxy-binding

### PR DESCRIPTION
**Pull request by stuartwdouglas**

### CODE REVIEW

**Comment by stuartwdouglas** [comment]
Retest this please

**Comment by n1hility** [comment]
retest this please

File: legacy/web/src/main/java/org/jboss/as/web/WebMigrateOperation.java [hunk 1]
```diff
@@ -791,11 +790,11 @@ private void migrateConnector(OperationContext context, Map<PathAddress, ModelNo
         addConnector.get(Constants.MAX_CONNECTIONS).set(newAddOp.get(WebConnectorDefinition.MAX_CONNECTIONS.getName()));
         addConnector.get(Constants.MAX_BUFFERED_REQUEST_SIZE).set(newAddOp.get(WebConnectorDefinition.MAX_SAVE_POST_SIZE.getName()));
         if(newAddOp.hasDefined(WebConnectorDefinition.REDIRECT_PORT.getName())) {
+            warnings.add(WebLogger.ROOT_LOGGER.couldNotMigrateResource(WebConnectorDefinition.PROXY_BINDING.getName(), pathAddress(newAddOp.get(ADDRESS))));
+        }
+        if(newAddOp.hasDefined(WebConnectorDefinition.PROXY_BINDING.getName())) {
             warnings.add(WebLogger.ROOT_LOGGER.couldNotMigrateResource(WebConnectorDefinition.REDIRECT_PORT.getName(), pathAddress(newAddOp.get(ADDRESS))));
         }
```
Comment (bstansberry) [inline legacy/web/src/main/java/org/jboss/as/web/WebMigrateOperation.java]: The attributes used in the warning messages don't align with the if statements -- they're reversed.

**Comment by bstansberry** [comment]
Superceded by #8359

**Pull request closed without merging**